### ISSUE TITLE
Add KeepassDX to Accrescent

### ISSUE THREAD

**Issue by guilamoff** [Kunzisoft/KeePassDX#1566]
Could you add your keepassDX application to the new Accrescent application store. 
Accrescent is an open-source, next generation application store and is very easy to use. Here is the link https://accrescent.app/features

Sincerely

**Comment by J-Jamet** [comment]
I can't even access the developer console because I need an invitation. If we have to be invited to upload an application, that's already a bad sign.

**Comment by realeatham** [comment]
Any updates?

**Comment by PrivacyAndSecurity** [comment]
I agree and would love to see you there and download the App from there.

Maybe you ask why we want this or what Benefits it would have for us as Users.

The Benefits for me would be a few, like these:

I could get KeePassDX from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

It would make it a lot easier for me and save me time, because KeePassDX is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

I really hope that you Guys could make that possible, because it would have great Benefits for us as Users. And think like that: If we support them by adding Apps, we help them to grow from Alpha to Stable.

**Comment by roamer4223** [comment]
I know this is on your to-do list so I hope this doesn't come across as pushy or annoying.

I just thought I'd mention that Accrescent is now also available directly from the built in GrapheneOS App store, so users of that particular OS can safely download it from there and access apps from the store safely and easily. I would also love to see you on there.

**Comment by Hasshu** [comment]
> Accrescent is now also available directly from the built in GrapheneOS App store

For context: https://grapheneos.social/@GrapheneOS/112821386750410102

**Comment by ghost** [comment]
I would love seeng both KeepassDX Libre & Free be the first password manager available on Accrescent just like Aves Gallery too :)

**Comment by midnight-wonderer** [comment]
@J-Jamet Google is blocking 3rd-party app stores soon.  
They created a form for public hearing; you might want to check this out:  
https://chaos.social/@grote/115095360358705273

**Comment by J-Jamet** [comment]
I wasn't aware of the controversy, but I think it's important to make people aware of the importance of computer freedom, and information is the best way to do that. After that, if we really can't install the programs we want, we'll have to bring out ethical devices for the general public, which are completely open source, in both hardware and software.
There will be aberrations, and we'll have to point them out to make people understand that choice is always beneficial, even in application ecosystem. And that trust is not something that can be imposed.

**Comment by ghost** [comment]
> I wasn't aware of the controversy, but I think it's important to make people aware of the importance of computer freedom, and information is the best way to do that. After that, if we really can't install the programs we want, we'll have to bring out ethical devices for the general public, which are completely open source, in both hardware and software. There will be aberrations, and we'll have to point them out to make people understand that choice is always beneficial, even in application ecosystem. And that trust is not something that can be imposed.

Will you reconsider adding support to Accrescent ? Their devs said that they will start looking into more app requests from developers later this year too :)

**Comment by J-Jamet** [comment]
I've never been against it, but the last time I looked it required a ticket, so I didn't go any further.
I just need to take the time to study the deployment platform.

**Comment by ghost** [comment]
> I've never been against it, but the last time I looked it required a ticket, so I didn't go any further. I just need to take the time to study the deployment platform.

That's great to hear. Accrescent overall is more secure and private for end users, more freedom and control for developets over their apps so i both side could really benefited from Accrescent in the future.

**Comment by ghost** [comment]
Accrescent is still in alpha and there have been issues with updating apps e.g. SunUp ad NextPush. Additionally installing beautyxt is currently broken. It would be wise to wait for Accrescent to reach stable to prevent such annoyances.
